### PR TITLE
fix: cut build time in half by removing redundant import for plugin data

### PR DESCRIPTION
**Pull request by shlomomdahan**
This PR cuts the build time in half.

There was a redundant import of the plugin versions data (~30mb) which this PR resolves.

While this does improve speed significantly, there are still further optimizations that should be made. 

ref: 
- #87

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -1,7 +1,22 @@
+// import { defineConfig } from 'vite'
+// import react from '@vitejs/plugin-react'
+
+// // https://vitejs.dev/config/
+// export default defineConfig({
+//     plugins: [react()],
+// })
+
```
Comment (krisstern) [inline vite.config.ts:8]:
```suggestion
```

**Review by krisstern — CHANGES_REQUESTED** [review]

**Review by Vandit1604 — APPROVED** [review]

**Review by krisstern — DISMISSED** [review]

**Comment by krisstern** [comment]
@gounthar Could you please review this PR for us?

**Review by krisstern — APPROVED** [review]

**Comment by krisstern** [comment]
@shlomomdahan There seems to be some issues with this PR:
```shell
[2024-07-16T14:19:57.254Z] + npm run lint
[2024-07-16T14:19:57.556Z] 
[2024-07-16T14:19:57.556Z] > stats2.jenkins.io@0.0.0 lint
[2024-07-16T14:19:57.557Z] > eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0
[2024-07-16T14:19:57.557Z] 
[2024-07-16T14:19:59.463Z] 
[2024-07-16T14:19:59.464Z] /home/jenkins/agent/workspace/Websites_stats_PR-91/src/hooks/useGetPluginVersionData.ts
[2024-07-16T14:19:59.464Z]   3:8  error  'rawPluginVersionData' is defined but never used  no-unused-vars
[2024-07-16T14:19:59.465Z]   3:8  error  'rawPluginVersionData' is defined but never used  @typescript-eslint/no-unused-vars
[2024-07-16T14:19:59.466Z] 
[2024-07-16T14:19:59.467Z] ✖ 2 problems (2 errors, 0 warnings)
[2024-07-16T14:19:59.468Z] 
```

**Review by gounthar — APPROVED** [review]
Looks good to me, thanks.

**Review by krisstern — APPROVED** [review]

**Pull request merged**